### PR TITLE
Race & Ethnicities: Sorting Races & Quick Fix

### LINKED ISSUE #180: [Frontend][Publisher] Race & Ethnicities Playtesting Followups
- Optimize saving
- Remove Back To Metrics link
- When disaggregations are disabled, grey-out grid area
- Make callout box clickable (opens the Race & Ethnicities right panel)
- Order Race & Ethnicities to have "Other" and "More than one race" as the bottom options
- Data Entry Publish Confirmation: Group By Ethnicity w/ Headers
- Update copy

### PR DESCRIPTION
**Pull request by mxosman**
## Description of the change

Quick last minute items:

- Almost forgot about sorting the races (sorted based on the updated spec):
Screenshot 2022-11-30 at 4 10 48 PM

- The active metric in the Settings menu metrics list lost its bottom blue border - it must've gotten lost somewhere in the overlapping PRs & rebases:
Screenshot 2022-11-30 at 4 11 53 PM

## Related issues

Closes #180

### Development

**This box MUST be checked by the submitter prior to merging**:
- [x] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/MetricConfiguration/RaceEthnicitiesGrid.tsx [hunk 1]
```diff
@@ -78,19 +79,21 @@ export const RaceEthnicitiesGrid: React.FC<{
...
-                <EthnicityCell
-                  key={ethnicity.key}
-                  enabled={ethnicity.enabled}
-                />
-              ))}
-            </EthnicitiesRow>
-          </RaceEthnicitiesRow>
-        ))}
+        {Object.entries(ethnicitiesByRace)
+          .sort(sortRaces)
```
Comment (mxosman) [inline publisher/src/components/MetricConfiguration/RaceEthnicitiesGrid.tsx:83]: L83 is the only change here.

File: publisher/src/components/MetricConfiguration/RaceEthnicitiesForm.tsx [hunk 2]
```diff
@@ -214,83 +215,87 @@ export const RaceEthnicitiesForm = observer(() => {
           )}
 
           {/* Races (Enable/Disable) */}
-          {Object.entries(ethnicitiesByRace).map(([race, ethnicities]) => {
-            const raceEnabled = Boolean(
-              Object.values(ethnicities).find((ethnicity) => ethnicity.enabled)
-            );
+          {Object.entries(ethnicitiesByRace)
+            .sort(sortRaces)
```
Comment (mxosman) [inline publisher/src/components/MetricConfiguration/RaceEthnicitiesForm.tsx:219]: L219 is the only change in this file.

**Comment by mxosman** [comment]
Despite it showing > 100 line changes - it's really only a small bit - a sorting function and a CSS line change.

**Review by lilidworkin — APPROVED** [review]
Awesome! Good eye

**Pull request merged**